packages/falcon-front-kit/src/DynamicRoute/DynamicRoute.tsx

```diff
@@ -51,3 +52,8 @@ export const DynamicRoute: React.SFC<DynamicRouteProps> = props => {
     </Router>
   );
 };
+DynamicRoute.propTypes = {
+  location: PropTypes.any,
+  components: PropTypes.any.isRequired,
```

Comment:
```suggestion
  components: PropTypes.objectOf(PropTypes.element).isRequired,
```
not sure about this one

Reply:
this is ok
```suggestion
components: PropTypes.objectOf(PropTypes.func.isRequired).isRequired,
```